lib/interface_quda.cpp

```diff
@@ -693,7 +693,7 @@ void initQuda(int dev)
 static cudaGaugeField* createExtendedGauge(cudaGaugeField &in, const int *R, TimeProfile &profile,
 					   bool redundant_comms=false, QudaReconstructType recon=QUDA_RECONSTRUCT_INVALID)
 {
-  // profile.TPSTART(QUDA_PROFILE_INIT);
+  profile.TPSTART(QUDA_PROFILE_INIT);
```

Comment:
Something looks wrong here.  You've added back the `TPSTART` but not the `TPSTOP`.  Intentional or bug?

Reply:
cb055928c